code/drasil-example/Drasil/GlassBR/TMods.hs

```diff
@@ -23,7 +23,7 @@ tMods = [pbIsSafe, lrIsSafe]
 lrIsSafe :: TheoryModel
 lrIsSafe = tm (EquationalModel lrIsSafeQD)
    [qw isSafeLoad, qw tmLRe, qw tmDemand] ([] :: [ConceptChunk])
-   [lrIsSafeQD] [sy isSafeLoad $= lrIsSafeExpr] [] [makeCite astm2009] 
+   [lrIsSafeQD] [] [] [makeCite astm2009] 
```

Comment:
This was duplicate information.

Reply:
Before one of my changes in this last commit, the first list of QDefs were unused in showing the equations, it completely relied on the list of invariants. So I have to remove the invariants from this since it, as an "invariant", was already given by lrIsSafeQD.